libglusterfs/src/inode.c

```diff
@@ -2654,7 +2657,8 @@ inode_ctx_size(inode_t *inode)
                 continue;
 
             xl = (xlator_t *)(long)inode->_ctx[i].xl_key;
-            old_THIS = THIS;
+            if (!old_THIS)
```

Comment:
should move to line 2674 (and can remove line 2669?) and fix line 2676 to be under the if statement as well.

Reply:
Done